inc/Config/DataSource/HttpDataSource.php

```diff
@@ -27,9 +27,6 @@ abstract public function get_display_name(): string;
 
 	abstract public function get_endpoint(): string;
 
-	/**
-	 * @inheritDoc
```

Comment:
Note these `@inheritDoc`s were removed due to a [failed phpcs check](https://github.com/Automattic/remote-data-blocks/actions/runs/12360487684/job/34495557377?pr=250):

```
$ composer run phpcs

FILE: tests/inc/Mocks/MockDataSource.php
--------------------------------------------------------------------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
--------------------------------------------------------------------------------------------------------------------------------------------------------
 36 | ERROR | [x] Useless documentation comment with @inheritDoc. (SlevomatCodingStandard.Commenting.UselessInheritDocComment.UselessInheritDocComment)
--------------------------------------------------------------------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
--------------------------------------------------------------------------------------------------------------------------------------------------------


FILE: inc/ExampleApi/Queries/ExampleApiDataSource.php
--------------------------------------------------------------------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
--------------------------------------------------------------------------------------------------------------------------------------------------------
 25 | ERROR | [x] Useless documentation comment with @inheritDoc. (SlevomatCodingStandard.Commenting.UselessInheritDocComment.UselessInheritDocComment)
--------------------------------------------------------------------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
--------------------------------------------------------------------------------------------------------------------------------------------------------


FILE: inc/Config/DataSource/HttpDataSource.php
--------------------------------------------------------------------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
--------------------------------------------------------------------------------------------------------------------------------------------------------
 30 | ERROR | [x] Useless documentation comment with @inheritDoc. (SlevomatCodingStandard.Commenting.UselessInheritDocComment.UselessInheritDocComment)
--------------------------------------------------------------------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
--------------------------------------------------------------------------------------------------------------------------------------------------------
```